api/src/common/decorators_api_views.py

```diff
@@ -0,0 +1,18 @@
+from django.utils.decorators import method_decorator
+from drf_yasg.utils import swagger_auto_schema
+
+# common decorators applied to all model view set classes
+def viewsets(tag):
```

Comment:
Also I think we change the name to reflect what this decorator is for. This is a decorator for adding the swagger auto schema decorators to model view sets.

Reply:
changed it to viewsets_swagger_decorator